saleor/graphql/schema.graphql

```diff
@@ -2151,7 +2398,7 @@ type Query {
   checkouts(before: String, after: String, first: Int, last: Int): CheckoutCountableConnection
   checkoutLine(id: ID): CheckoutLine
   checkoutLines(before: String, after: String, first: Int, last: Int): CheckoutLineCountableConnection
-  addressValidator(input: AddressValidationInput!): AddressValidationData
+  addressValidator(countryCode: CountryCode, countryArea: String, cityArea: String): AddressValidationData
```

Comment:
I would rename it to `addressValidationRules` so it sounds more natural as a query name.